tutorials/vm-migration-with-stratozone/index.md

```diff
@@ -1,42 +1,45 @@
 ---
-title: Migrate on-premises virtual machines with StratoZone and Velostrata
-description: Learn how to use StratoZone and Velostrata to migrate on-premises VMs to Google Cloud Platform.
+title: Migrate on-premises virtual machines with StratoZone and Migrate for Compute Engine
+description: Learn how to use StratoZone and Migrate for Compute Engine to migrate on-premises VMs to Google Cloud Platform.
 author: laidig
-tags: VM migration, Velostrata, StratoZone
+tags: VM migration, Migrate for Compute Engine, StratoZone, Migrate for Compute Engine
```

Comment:
You both added and changed a tag here. I'd undo the changed one, so that people searching for 'Velostrata' find this.

Reply:
Done.